<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-D4G96PPV" ignored="0" name="DDT_Registration" tests="7" failures="7" timestamp="2020-09-18T00:57:54 EDT" time="16.763" errors="0">
  <testcase name="verifyEmail" time="1.959" classname="testcases.TC020_InvalidData">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testcases.TC020_InvalidData.verifyEmail(TC020_InvalidData.java:58)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyEmail -->
  <testcase name="verifyEmail" time="1.759" classname="testcases.TC020_InvalidData">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testcases.TC020_InvalidData.verifyEmail(TC020_InvalidData.java:58)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyEmail -->
  <testcase name="verifyEmail" time="1.771" classname="testcases.TC020_InvalidData">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testcases.TC020_InvalidData.verifyEmail(TC020_InvalidData.java:58)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyEmail -->
  <testcase name="verifyEmail" time="2.217" classname="testcases.TC020_InvalidData">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testcases.TC020_InvalidData.verifyEmail(TC020_InvalidData.java:58)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyEmail -->
  <testcase name="verifyEmail" time="1.273" classname="testcases.TC020_InvalidData">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testcases.TC020_InvalidData.verifyEmail(TC020_InvalidData.java:58)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyEmail -->
  <testcase name="verifyEmail" time="1.811" classname="testcases.TC020_InvalidData">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testcases.TC020_InvalidData.verifyEmail(TC020_InvalidData.java:58)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyEmail -->
  <testcase name="verifyEmail" time="1.665" classname="testcases.TC020_InvalidData">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testcases.TC020_InvalidData.verifyEmail(TC020_InvalidData.java:58)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyEmail -->
</testsuite> <!-- DDT_Registration -->
